gfx_scene.h

```diff
@@ -1152,7 +1152,7 @@ class GfxSceneInternal
                     animation_channel.values_[4 * next_index + 1], animation_channel.values_[4 * next_index + 2]);
                 animated_node->rotate_ = glm::slerp(previous, next, interpolate);
             }
-            if(animation_channel.type_ == kGltfAnimationChannelType_Scale)
+            else if(animation_channel.type_ == kGltfAnimationChannelType_Scale)
```

Comment:
Without this 'else if' then the code will execute the proceeding 'else' branch which is specific to animations of type `kGltfAnimationChannelType_Weights`. However without this fixe it was being executed for all animation types other the 'scale' which was just generating erroneous data.